src/main/java/uk/gov/hmcts/reform/pip/subscription/management/repository/SubscriptionRepository.java

```diff
@@ -31,6 +31,13 @@ public interface SubscriptionRepository extends JpaRepository<Subscription, Long
     List<Subscription> findSubscriptionsBySearchValue(@Param("search_type") String searchType,
                                                       @Param("search_value") String searchValue);
 
```

Comment:
Bug below from testing - court_name should be location_name. Currently producing exception